src/textual/widget.py

```diff
@@ -1348,10 +1349,13 @@ def scroll_to(
             duration (float | None, optional): Duration of animation, if animate is True and speed is None.
             easing (EasingFunction | str | None, optional): An easing method for the scrolling animation. Defaults to "None",
                 which will result in Textual choosing the default scrolling easing function.
+            force (bool,optional): Force scrolling even when prohibited by overflow styling. Defaults to `False`.
```

Comment:
```suggestion
            force (bool, optional): Force scrolling even when prohibited by overflow styling. Defaults to `False`.
```

There are a bunch of these, so maybe Search & Replace and then a single commit? 🤷